Library/Homebrew/utils/analytics.rb

```diff
@@ -290,10 +291,21 @@ def custom_prefix_label(verbose: false)
       def arch_label(verbose: false)
         if Hardware::CPU.arm?
           "ARM"
+        elsif verbose
+          "x86_64"
         else
           ""
         end
       end
+      alias generic_arch_label arch_label
+
+      sig { returns(String) }
+      def homebrew_version
+        version = HOMEBREW_VERSION.match(/^([\d.]*)-?/)[1]
```

Comment:
@SMillerDev What's this look like for normal versions vs. developer versions?

Reply:
It's either 4.0.0 or 4.0.0-dev